backend/amd64/selection.ml

```diff
@@ -278,6 +285,16 @@ method! select_operation op args dbg =
       | ("caml_int64_crc_unboxed", [|Int|]
         | "caml_int_crc_untagged", [|Int|]) when !Arch.crc32_support ->
           Ispecific Icrc32q, args
+      | "caml_float_iround_half_to_even_unboxed", [|Int|] ->
+         Ispecific Ifloat_iround, args
+      | "caml_float_min_unboxed", [|Float|] ->
+         (* swap the order of arguments to match C stubs behavior on Nan *)
+         let arg0, arg1 = two_args "minsd" args in
```

Comment:
(Not important, but since the only uses of `two_args` are to
get a couple and build a list from it, why not simply have
a `swap_xyz` function that would return the list?)

Reply:
I've finally managed to fix this properly, turns out this swap is not needed at all. 
I had a bug in the emitter, used arg 0 instead of arg1, quite embarrassing..

Reply:
Then the `two_args` function should probably go.

Reply:
ah, yes - thanks!